### PR TITLE
Pulls in changes from upstream sys.config as of 2022.01.12.1

### PR DESCRIPTION
**Pull request by vpetersson**
**Issue**

We're seeing an increased amount of `failed to dial challenger` in miner logs, which affects earnings.

**How**
* Manually diffed `sys.config` from the GA release on 2022.01.12.1 with our template.

### CODE REVIEW

**Review by KevinWassermann94 — APPROVED** [review]

**Review by SebastianMaj — APPROVED** [review]

File: config.template [hunk 1]
```diff
@@ -102,10 +105,11 @@
    {block_time, 60000},
    {late_block_timeout_seconds, 3600},
    {update_dir, "/opt/miner/update"},
```
Comment (marvinmarnold) [inline config.template:107]: Missing `{hotfix_dir, "/opt/miner/hotfix"},` https://github.com/helium/miner/blame/master/config/sys.config#L106

File: config.template [hunk 2]
```diff
@@ -102,10 +105,11 @@
...
+   {api_base_url, "https://api.helium.io/v1"},
```
Comment (marvinmarnold) [inline config.template:108]: Should we change to stakejoy endpoint?

**Comment by shawaj** [comment]
Need to update the test case of sample_output.txt but note the but in the README about trimming the last character

File: config.template [hunk 3]
```diff
@@ -54,7 +55,7 @@
    {base_dir, "/var/data"},
    {onboarding_dir, "/mnt/uboot"},
    {num_consensus_members, 16},
-   {seed_nodes, "/ip4/18.217.27.26/tcp/2154,/ip4/35.161.222.43/tcp/443,/ip4/99.80.158.114/tcp/2154"},
+   {seed_nodes, "/ip4/18.217.27.26/tcp/2154,/ip4/35.161.222.43/tcp/443,/ip4/99.80.158.114/tcp/2154,/ip4/3.66.43.167/tcp/443,/ip4/52.220.121.45/tcp/2154,/ip4/54.207.252.240/tcp/443,/ip4/3.34.10.207/tcp/2154,/ip4/13.238.174.45/tcp/443"},
    {peerbook_update_interval, 900000},
    {max_inbound_connections, 6},
    {outbound_gossip_connections, 2},
```
Comment (marvinmarnold) [inline config.template:61]: Missing `{gossip_version, 2},` https://github.com/helium/miner/blame/master/config/sys.config#L62

**Review by marvinmarnold — CHANGES_REQUESTED** [review]
We should add `gossip_version` and `hotfix_dir`. Can try changing `api_base_url` in future to be consistent with what they have for nw.

**Comment by shawaj** [comment]
Just a quick thought. Shall we add a workflow to automatically diff our config against upstream and notify us of any changes?

**Comment by KevinWassermann94** [comment]
@shawaj I think that might be a good idea

**Comment by shawaj** [comment]
@vpetersson ... On the text file https://github.com/NebraLtd/hm-block-tracker/pull/62#issuecomment-1022239645

File: config.template (hunk 2 above)
Comment (vpetersson) [reply config.template:108]: Yeah good shout. Let's do that.

**Comment by vpetersson** [comment]
> Just a quick thought. Shall we add a workflow to automatically diff our config against upstream and notify us of any changes?

Yeah but let's get this out first and make sure it is the root cause.

File: config.template (hunk 2 above)
Comment (vpetersson) [reply config.template:109]: On a second thought -- let's not do that as it's a 'Secret URL'.

**Review by marvinmarnold — APPROVED** [review]

**Review by ilyastrodubtsev — APPROVED** [review]

**Pull request merged**